packages/generator/src/edmx-to-vdm/v4/operation.ts

```diff
@@ -124,13 +124,31 @@ export function filterAndTransformOperations(
     operation => (operation.IsBound.toLowerCase() === 'true') === isBound
```

Comment:
[req] This name is confusing. It sounds like those are bound operations, but it could be either bound or unbound. How about `filteredOperations` or `relevantOperations`?

BTW, last time we talked about this we assumed this was only about bound operations, therefore we might have incorrect assumptions in the following.

Reply:
Okay this makes sense. It also makes sense for `splitOperationsByMissingImports` to run before this because we are only using `relevantOperations` to filter out all bound operations without a parameter and all operations without an operationImport are not required for this. I believe it would make sense for `relevantOperations` to only filter those operations with an operationImport. So something like:

```javascript
const [withoutOperation, withOperation] = splitOperationsByMissingImports(
  operationImports,
  operations
);

const relevantOperations = withOperation.filter(
  operation => (operation.IsBound) === isBound
);
```

Reply:
Okay there was a test case that checks for operations referenced by operation import. The probem was that the messageContext in the `createLogger` method was wrong. It should be 'operation' instead of 'action-import'. Making this change results in the test failing on the old state. 

The way to fix it is to make sure that we are only using `relevantOperations` to filter out all bound operations without a parameter. Operations without an operationImport are not required for this so it should not be related with `splitOperationsByMissingImports`. I believe it would make sense for `relevantOperations` to only filter those operations with an operationImport. I have implemented it this way.